tests/expectations/tests/381-decltype-alias.rs

```diff
@@ -10,3 +10,6 @@ pub struct std_allocator_traits<_Alloc> {
     pub _address: u8,
     pub _phantom_0: ::std::marker::PhantomData<_Alloc>,
 }
+impl <_Alloc> Default for std_allocator_traits<_Alloc> {
```

Comment:
Yes, it's hard to trace which type could be used in where, such as  `base_members`, `template_args`, `fields` or `ref_template`. So, the most straight way is to implement `Default` as much as possible.